doc/syntax/refinements.rdoc

```diff
@@ -212,10 +212,7 @@ all refinements from the same module are active when a refined method
 
 When looking up a method for an instance of class +C+ Ruby checks:
 
-* If refinements are active for +C+, in the reverse order they were activated:
-  * The prepended modules from the refinement for +C+
-  * The refinement for +C+
-  * The included modules from the refinement for +C+
+* If refinements are active for +C+, in the reverse order they were activated
```

Comment:
This is just a nit, and really outside the scope of the PR correction, but I wonder about rewording to:

> The refinements of +C+, in reverse order of activation

A reword could also be done in a separate PR and I don't think should block a fix. Totally agree with removing include/prepend order mention since it's no longer applicable and potentially confusing.

Good catch! 👏